xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/Inputstream.h

```diff
@@ -121,28 +150,83 @@ extern "C" {
       FLAG_VISUAL_IMPAIRED = 0x0100
     };
 
-    enum INPUTSTREAM_COLORSPACE
-    {
-      COLORSPACE_UNKNOWN,
-      COLORSPACE_BT709,
-      COLORSPACE_BT470M,
-      COLORSPACE_BT470BG,
-      COLORSPACE_SMPTE170M,
-      COLORSPACE_SMPTE240M,
-      COLORSPACE_FILM,
-      COLORSPACE_BT2020,
-      COLORSPACE_SMPTE428,
-      COLORSPACE_SMPTEST428_1,
-      COLORSPACE_SMPTE431,
-      COLORSPACE_SMPTE432,
-      COLORSPACE_JEDEC_P22
+    //Keep in sync with AVColorSpace
+    enum COLORSPACE
+    {
+      COLORSPACE_RGB = 0,
+      COLORSPACE_BT709 = 1,
+      COLORSPACE_UNSPECIFIED = 2,
+      COLORSPACE_UNKNOWN = COLORSPACE_UNSPECIFIED, // compatibility
+      COLORSPACE_RESERVED = 3,
+      COLORSPACE_FCC = 4,
+      COLORSPACE_BT470BG = 5,
+      COLORSPACE_SMPTE170M = 6,
+      COLORSPACE_SMPTE240M = 7,
+      COLORSPACE_YCGCO = 8,
+      COLORSPACE_YCOCG = COLORSPACE_YCGCO,
+      COLORSPACE_BT2020_NCL = 9,
+      COLORSPACE_BT2020_CL = 10,
+      COLORSPACE_SMPTE2085 = 11,
+      COLORSPACE_CHROMA_DERIVED_NCL = 12,
+      COLORSPACE_CHROMA_DERIVED_CL = 13,
+      COLORSPACE_ICTCP = 14,
+      COLORSPACE_MAX
     };
 
-    enum INPUTSTREAM_COLORRANGE
+    //Keep in sync with AVColorPrimaries
+    enum COLORPRIMARIES : int32_t {
+      COLORPRIMARY_RESERVED0 = 0,
+      COLORPRIMARY_BT709 = 1,
+      COLORPRIMARY_UNSPECIFIED = 2,
+      COLORPRIMARY_RESERVED = 3,
+      COLORPRIMARY_BT470M = 4,
+      COLORPRIMARY_BT470BG = 5,
+      COLORPRIMARY_SMPTE170M = 6,
+      COLORPRIMARY_SMPTE240M = 7,
+      COLORPRIMARY_FILM = 8,
+      COLORPRIMARY_BT2020 = 9,
+      COLORPRIMARY_SMPTE428 = 10,
+      COLORPRIMARY_SMPTEST428_1 = COLORPRIMARY_SMPTE428,
+      COLORPRIMARY_SMPTE431 = 11,
+      COLORPRIMARY_SMPTE432 = 12,
+      COLORPRIMARY_JEDEC_P22 = 22,
+      COLORPRIMARY_MAX
+    };
+
+    //Keep in sync with AVColorRange
+    enum COLORRANGE
     {
       COLORRANGE_UNKNOWN,
       COLORRANGE_LIMITED,
-      COLORRANGE_FULLRANGE
+      COLORRANGE_FULLRANGE,
+      COLORRANGE_MAX
+    };
+
+    // keep in sync with AVColorTransferCharacteristic
+    enum COLORTRC : int32_t
+    {
+      COLORTRC_RESERVED0    = 0,
+      COLORTRC_BT709        = 1,
+      COLORTRC_UNSPECIFIED  = 2,
+      COLORTRC_RESERVED     = 3,
+      COLORTRC_GAMMA22      = 4,
+      COLORTRC_GAMMA28      = 5,
+      COLORTRC_SMPTE170M    = 6,
+      COLORTRC_SMPTE240M    = 7,
+      COLORTRC_LINEAR       = 8,
+      COLORTRC_LOG          = 9,
+      COLORTRC_LOG_SQRT     = 10,
+      COLORTRC_IEC61966_2_4 = 11,
+      COLORTRC_BT1361_ECG   = 12,
+      COLORTRC_IEC61966_2_1 = 13,
+      COLORTRC_BT2020_10    = 14,
+      COLORTRC_BT2020_12    = 15,
+      COLORTRC_SMPTE2084    = 16,
+      COLORTRC_SMPTEST2084  = COLORTRC_SMPTE2084,
+      COLORTRC_SMPTE428     = 17,
```

Comment:
remove the vertical alignment in the whole block, https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#35-vertical-alignment